test-integration/simple-project-truffle-web3-1.2.0/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+
+cd "$(dirname $0)"
+
+cp -r ../simple-project-truffle-template .
+
+sed -i "s/\"truffle\": \"5.0.1\"/\"truffle\": \"5.0.29\"/g" simple-project-truffle-template/package.json
```

Comment:
I really think it would be best to simply use `npm install`.

```suggestion
cd simple-project-truffle-template
npm install truffle@5.0.29
```